rust/minimal-example/README.md

```diff
@@ -0,0 +1,67 @@
+# Readme
+
+This project is an example of how to create a cross-platform Arma extension in Rust. 
+The extension has no dependencies except on the core Rust libraries, and builds into a single .dll.
+
+The extension will run on Linux and Windows, both 32-bit and 64-bit.
+
+It supports:
+* RVExtensionVersion
+* RVExtension
+* RVExtensionArgs
+* RVExtensionRegisterCallback
+
+## Author
+
+Spoffy (https://github.com/Spoffy/Rust-Arma-Extension-Example)
+
```

Comment:
Please insert a `How to setup` section (see https://github.com/Spoffy/RV-Extension-Examples/tree/rust-example/cs/net%20core#how-to-setup for reference).

Reply:
Will do, I'll add it tomorrow.